test/test_decomp.py

```diff
@@ -347,6 +347,7 @@ def normalize_op_input_output(f, sample, requires_grad=True):
     # Decomposed backward formula is not as precise
     ("cpu", torch.bfloat16, "nn.functional.hardswish"),
     ("cuda", torch.float16, "nn.functional.cross_entropy"),
+    (None, None, "nn.functional.rrelu"),
```

Comment:
I don't actually think this is the right way to fix this test. Without this exclusion, I see this:

```
...
  File "/home/slarsen/local/pytorch/test/test_decomp.py", line 676, in do_cross_ref
    check_decomposed(op.aten_backward_name, mode)
  File "/home/slarsen/local/pytorch/test/test_decomp.py", line 641, in check_decomposed
    self.assertTrue(
AssertionError: False is not true : aten.rrelu_with_noise_backward was not decomposed, saw calls for: aten.add.Tensor, aten.where.self, aten.is_same_size.default, aten.mul.Tensor, aten.zeros_like.default, aten.scalar_tensor.default. If your op is  CompositeImplicitAutograd you should skip this test by updating CROSS_REF_EXCLUDE_SET.
```

Reply:
> If your op is CompositeImplicitAutograd

we're making it CompositeImplicitAutograd by making the decomp run before autograd.. So maybe `CROSS_REF_EXCLUDE_SET` is correct ?